Dockerfile

```diff
@@ -8,14 +8,15 @@ FROM ${BASE_IMAGE_NAME}:${BASE_IMAGE_TAG}
 # ... then the modules installed during the transformation are not loaded automatically!
 # ... because the loader thinks that this directory does not exist (it did not exist at the start of the script)
 # Eg. mkdir -p /var/www/.local/lib/python3.8/site-packages
-RUN mkdir -p $(su www-data -s /bin/bash -c "python -c 'import site; print(site.USER_SITE)'")
+RUN . $VIRTUAL_ENV"/bin/activate" && mkdir -p $(su www-data -s /bin/bash -c "python -c 'import site; print(site.USER_SITE)'")
 
 # Make home directory writable
 RUN chown -R www-data:www-data /var/www
 
 WORKDIR /code/
-COPY ./src ./src
-COPY ./tests ./tests
-COPY main.py README.md ./
+COPY . .
```

Comment:
Personally I prefer the original way. Now you must add anything, that should not be in the image, to `.dockerignore` (`azure-pipelines/` for example) and always think about it when creating anythong new in the root.